stream-chat-android-docs/src/main/java/io/getstream/chat/docs/java/Android.java

```diff
@@ -714,6 +718,49 @@ public void getSyncHistory(ChatClient chatClient) {
         }
     }
 
+    /**
+     * @see <a href="https://getstream.io/nessy/docs/chat_docs/unread_counts/unread_channel">Channels</a>
+     */
+    public class UnreadCount extends Fragment {
+
+        public void getUnreadCountInfo() {
+            // Get channel
+            Channel channel = ChatDomain.instance()
+                    .getUseCases()
```

Comment:
We are going to deprecate `ChatDomain.useCases` soon in https://github.com/GetStream/stream-chat-android/pull/1698.
Maybe you could update it to the new API when it's merged?